pkg/notification/manager/sql/sql.go

```diff
@@ -163,6 +164,44 @@ func (m *mgr) GetNotification(ref string) (*notification.Notification, error) {
 	return &n, nil
 }
 
+func (m *mgr) GetNotificationPreference(opaqueId string) (bool, error) {
+	query := `
+	SELECT n.configvalue
+	FROM oc_preferences AS n
+	WHERE n.userid = ?
+	AND n.appid = 'disableNotificationsCore'
+	AND n.configkey = 'disableNotifications'
+	`
+	rows, err := m.db.Query(query, opaqueId)
+	if err != nil {
+		return false, err
+	}
+
+	defer rows.Close()
+
+	count := 0
+	var configvalues []bool
+
+	for rows.Next() {
```

Comment:
You can query for a single row, take a look [here](https://github.com/javfg/reva/blob/a80dc6ffc4d4e31f5218933ab8d9d6a58956faf8/pkg/cbox/preferences/sql/sql.go#L93).